index.js

```diff
@@ -110,7 +110,10 @@ module.exports = (app, options) => {
         })
 
         const contentType = (res.headers['content-type'] || res.headers['Content-Type'] || '').split(';')[0]
-        const isBase64Encoded = options.binaryMimeTypes.indexOf(contentType) > -1
+        const contentEncoding = res.headers['content-encoding'] || res.headers['Content-Encoding']
+        const isCompressed = contentEncoding && contentEncoding !== 'identity'
+        const customBinaryCheck = typeof options.enforceBase64 === 'function' && options.enforceBase64(res) === true
+        const isBase64Encoded = options.binaryMimeTypes.indexOf(contentType) > -1 || isCompressed || customBinaryCheck
```

Comment:
```suggestion
        const isBase64Encoded = options.binaryMimeTypes.indexOf(contentType) > -1 || isCompressed(res) || customBinaryCheck(options,res)
```
Wouldn't it be better to have function calls instead of interpreting the result before? Technically if `contentType` is properly set you're calculating `isCompressed` and `customBinaryCheck` even if it's not used

Reply:
That's completely reasonable. Changed to use functions.